### PR TITLE
[DO NOT MERGE] Import EdgeX v2.0.0

### PR DESCRIPTION
**Pull request by t25kim**
# Description

This is a temporary branch for testing edgex v2.0.0 against DataStorage.

WIP #324 

## Issue 
EdgeX uses 0mq which is a shared library and makes cross-compilation difficult.

## Type of change

- [x] Code cleanup/refactoring
- [x] This change requires a documentation update

# How Has This Been Tested?
See https://github.com/lf-edge/edge-home-orchestration-go/issues/324#issuecomment-873858913
```

### CODE REVIEW

**Comment by tdrozdovsky** [comment]
Thank you! I think the Go version should also be changed here:
```
docs/platforms/orange_pi3/orange_pi3.md:72:- go compiler (install a version not lower than 1.12.5)
docs/platforms/raspberry_pi3/raspberry_pi3.md:70:- go compiler (install a version not lower than 1.12.5, recommended v1.14.x)
docs/platforms/x86_64_linux/x86_64_linux.md:40:- go compiler
```
It is also possible to exclude the build possibility of edge-orchestration directly on boards (orange_pi3 and raspberry_pi3 ) Since it may be difficult to support Go newest version. What do you think about it?

File: deployments/datastorage/docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#  * Copyright 2020 Intel Corporation.
```
Comment (tdrozdovsky) [inline deployments/datastorage/docker-compose.yml:1]: Are you sure you need to change the year of issue?

Comment (t25kim) [reply deployments/datastorage/docker-compose.yml:1]: I pasted the orignal [code](https://github.com/edgexfoundry/edgex-compose/blob/main/docker-compose-no-secty.yml#L1).

Comment (tdrozdovsky) [reply deployments/datastorage/docker-compose.yml:1]: Ok, Great!

**Comment by t25kim** [comment]
> Thank you! I think the Go version should also be changed here:
> 
> ```
> docs/platforms/orange_pi3/orange_pi3.md:72:- go compiler (install a version not lower than 1.12.5)
> docs/platforms/raspberry_pi3/raspberry_pi3.md:70:- go compiler (install a version not lower than 1.12.5, recommended v1.14.x)
> docs/platforms/x86_64_linux/x86_64_linux.md:40:- go compiler
> ```

Thanks for the information!

> 
> It is also possible to exclude the build possibility of edge-orchestration directly on boards (orange_pi3 and raspberry_pi3 ) Since it may be difficult to support Go newest version. What do you think about it?

We build the edge-orchestration in terms of OS architecture rather than development boards, and I think we don't need to remove it at this time because golang supports different OS architectures. I'd like to talk about this issue when we encounter this problem.

**Comment by t25kim** [comment]
EdgeX v2.1(Jakarta) has been released. We will think of what version of edgex should be taken.

**Pull request closed without merging**